index.html

```diff
@@ -201,6 +229,18 @@ <h1 class="primary-text">Get started</h1>
     </div>
   </footer>
 
+  <footer class="flex-column">
```

Comment:
- [ ] Please I highly recommend working on your `footer` section due to duplicates of it. 

_**REQUIRED FIGMA DESIGN**_

![Screenshot (209)](https://user-images.githubusercontent.com/78104544/230528837-47c6743e-5725-42b8-b25e-7268232953c0.png)

_**YOUR DESIGN**_

![Screenshot (211)](https://user-images.githubusercontent.com/78104544/230528871-476a2605-24d2-4957-b3d2-5c6506e7ac71.png)